ui/main/src/app/services/cards/server/AngularCardsServer.ts

```
@@ -21,7 +21,7 @@ import {CardForPublishing} from '../model/CardForPublishing';
 import {CardWithChildCards} from '../model/CardWithChildCards';
 import {CardCreationReportData} from '../model/CardCreationReportData';
 import {FieldToTranslate} from '@ofServices/cards/model/FieldToTranslate';
-import {LightCard} from 'app/model/LightCard';
+import {Card} from 'app/model/Card';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Card'. Consider to delete it or rename it.

'Card' is defined here:


(lint/suspicious/noRedeclare)